website-guidelines.md

```diff
@@ -38,7 +38,9 @@ sandbox project.” and the CNCF logo. We also appreciate a link to KubeCon + Cl
 events approach.
 7. Website footers must include trademark guidelines by either linking to [Trademark Usage][]
 (directly or via a "Terms of service" page), or by including the following text: "The Linux Foundation® (TLF)
-has registered trademarks and uses trademarks. For a list of TLF trademarks, see [Trademark Usage][]".
+has registered trademarks and uses trademarks. For a list of TLF trademarks, see [Trademark Usage][]". 
+If your project has been converted to the Series LLC model (starting in 2025) please use the following 
```

Comment:
Not sure if suggestions are welcome (has this language already been vetted by legal folks?), but I think we should add a comma.
```suggestion
If your project has been converted to the Series LLC model (starting in 2025), please use the following 
```